common/src/main/java/dev/lavalink/youtube/clients/skeleton/NonMusicClient.java

```diff
@@ -87,8 +87,8 @@ protected JsonBrowser loadTrackInfoFromInnertube(@NotNull YoutubeAudioSourceMana
 
         ClientConfig config = getBaseClientConfig(httpInterface);
 
-        if (status == null) {
-            // Only add embed info if the status is not NON_EMBEDDABLE.
+        if (status == null || isEmbedded()) {
+            // Only add embed info if the status and client are embeddable..
             config.withClientField("clientScreen", "EMBED")
```

Comment:
This field is what allows certain age-restricted videos to be playable on embed clients.